_docs/style/colors.md

```diff
@@ -3,28 +3,157 @@ title: Color Palette
+
+- Do not add colors to the color palette or the color families.
+- Be intentional about using color families. Don't use colors outside of a color family for a component that belongs to the family.
+- When you apply a color to an object, consider all of the other colors used on the page. Because color commands the attention of the user, ensure that the colors you use get the right amount of attention, regardless of the data set, device type, or permission setting.
+- Use all of the interactive states for an interactive object.
+
+<!---
+
+- Do not use multiple color families in data visualizations.
+
+--->
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Accessibility
+
+The colors you select when you construct a user interface (UI) have a large impact on the accessibility of the UI for color blind and low vision users. When choosing colors, ensure that you consider the guidelines described in [Accessibility]({{site.baseurl}}/principles/accessibility.html).
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Helix color families
+
+Helix color families streamline the large number of colors into groups that you use for specific purposes. For example, when you want to draw the attention of the user to an object, use the health status or object status colors.
+
+Color families also provide a predetermined set of colors that you can use when you design a new component. For example, if you create an interactive element, use the interactive states color family.
+
+Helix provides the following color families:
+
+- [Health status](#health-status)
+- [Object status](#object-status)
+- [Interactive states](#interactive-states)
+- [Application frame](#application-frame)
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Health status
+
+The health status family communicates the well-being of an infrastructure object,
+for example, a load balancer or a server, or an entity, for example, an account
+or a user.
+
+The health status color family has strict usage guidelines.
+
+- Use the 500 hue as the default health status colors (red, yellow, green).
+- Use the 700 hue for hover state (red, yellow, green).
+
+Refer to [Iconography]({{site.baseurl}}/style/iconography) for guidelines about constructing default and hover state health status icons.
+
+- Use the 100 hue (red, yellow, green) for background colors that you can apply to components such as the table heartbeat and a [Message Banner]({{site.baseurl}}/components/message-banner.html).
```

Comment:
The message banner switch isn't on this version of the site, so this link is broken.